bin/damsolrizer

```diff
@@ -101,12 +101,23 @@ begin
       max_tries = 2
       puts @msg.headers.inspect
       puts "PID: #{@msg.headers["pid"]}"
+      puts "method: #{method}"
       begin
-        if ["addDatastream", "addRelationship","ingest","modifyDatastreamByValue","modifyDatastreamByReference","modifyObject","purgeDatastream","purgeRelationship"].include? method
+        if ["modifyCollection", "addDatastream", "addRelationship","ingest","modifyDatastreamByValue","modifyDatastreamByReference","modifyObject","purgeDatastream","purgeRelationship"].include? method
           solrizer = Solrizer::Fedora::Solrizer.new
           solrizer.solrize @msg.headers["pid"], solropts
```

Comment:
@hweng For your comment "@lsitu The order would be first the solr indexing got updated, then it call Dams::ShareApi.share_push.
The autoCommit does a hard commit to the disk. Let me know if there is any advantage you would think to use it in solr indexing?" --
I think currently setting of the Solrizer should be indexing with a soft commit. So I am wondering whether it's will do a hard commit for you or not. If it does a hard commit here, then the Solrizer is misconfigured, which slow down the indexing process a lot. So "objectModify" type and "collectionModify" type should be done in the other way with solropts={:autocommit => true | false}.